returnn/util/file_cache.py

```diff
@@ -598,10 +598,10 @@ def _all_parent_dirs(filename: str, *, base_dir: str) -> List[str]:
     return dirs
 
 
-def _files_with_parents(filenames: Iterable[str], *, base_dir: str) -> Dict[str, bool]:
-    res = {}  # dict to have order deterministic
+def _files_with_parents(filenames: Iterable[str], *, base_dir: str) -> Dict[str, int]:
```

Comment:
I don't understand. Why is it relevant to return the count, and not just a bool?